### ISSUE TITLE
Text change on Type of deposit page in form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#590]
Please change the text at the top of the page to an H2: "What type of content will you deposit?"
Also, the "Which of the following terms further describe your deposit?" should be bold Paragraph.

**Comment by justinlittman** [comment]
@astridu Is this the correct heading?

<img width="726" alt="Image" src="https://github.com/user-attachments/assets/9c71d7c5-7560-4ff1-8613-3d6c5fe7ddfd" />

**Comment by astridu** [comment]
Yes, thank you!
